resources/views/clubevent/createClubEventView.blade.php

```diff
@@ -1,54 +1,96 @@
 @extends('layouts.master')
 
 @section('title')
-	{{ trans('mainLang.createNewVEvent') }}
+        @if($createClubEvent)
+        	{{ trans('mainLang.createNewVEvent') }}
```

Comment:
"VEvent"? I think that was a typo, wasn't it?

Reply:
didn't changed the translations, just restructured it, this translation is already in production